config/environment_indicator.switcher.feature.yml

```diff
@@ -0,0 +1,11 @@
+uuid: 6dd1db40-7b38-4361-a3b8-ff4d701ef5bf
+langcode: en
+status: true
+dependencies: {  }
+machine: feature
+description: null
+name: Feature
+weight: '6'
+url: 'https://feature.commondesign-unocha-org.ahconu.org'
+fg_color: '#db7b18'
+bg_color: '#ffffff'
```

Comment:
```suggestion
fg_color: '#ffffff'
bg_color: '#db7b18'
```

I believe the foreground/background got switched here. I've manually applied in my browser and took the screenshot below. Does it still provide enough distinction for you to see the difference, @lazysoundsystem ?

<img width="471" alt="env-indicator-colors" src="https://user-images.githubusercontent.com/254753/221504789-20d8dc68-3a60-4a09-8006-0f2e86e6df52.png">

Reply:
Thanks @rupl - good catch, and the difference between the colors is nice and clear to me. I've updated the PR